repaygithub/cactus
# CACTUS-264 :: Fix broken website links

Ticket: https://repayonline.atlassian.net/browse/CACTUS-264

### Reminder of Links to Be Fixed

- The “design language” link on the home page is broken. I think it might need to be reverted to an absolute link rather than a relative link.
- The “API Documentation” and “Feature Flag” links on the Framework home page are broken.
- The “Feature Flags” link on the Frameworks API Documentation page is broken.
- The “Theme” and “API Documentation” links on the Cactus Web homepage are broken.
- The “Available Icons” link on the Icons homepage is broken.
- All the links on the Internationalization homepage are broken.
- The links on the storybook home page are broken

*Note: I deleted the design language link.  The only way to fix it is to upgrade Gatsby, but upgrading Gatsby breaks a bunch of other things.  Unfortunately I think this is the best solution for now (there's a link to the page on the website side bar, so I don't think it's a big deal).*

### How to Test
1. Run yarn install in the root, as I had to upgrade some Gatsby packages
2. I had to publish the website to test that the double prefix issue was fixed (the package upgrades fixed it). So feel free to click around the published website to make sure all of the links listed above are now working. Normally I might not have published from a branch, but given how broken the currently published site was, anything I made was an improvement 😄
3. Click around the docs inside the GitHub repo on this branch and make sure the links work there too.
